tests/lib/rules/no-async-actions.js

```diff
@@ -0,0 +1,96 @@
+'use strict';
+
+//------------------------------------------------------------------------------
+// Requirements
+//------------------------------------------------------------------------------
+
+const rule = require('../../../lib/rules/no-async-actions.js');
+const RuleTester = require('eslint').RuleTester;
+
+
+//------------------------------------------------------------------------------
+// Tests
+//------------------------------------------------------------------------------
+
+const message = 'Do not use async actions';
+
+const ruleTester = new RuleTester({
+  parserOptions: {
+    ecmaVersion: 2018,
+    sourceType: 'module',
+  }
+});
+
+ruleTester.run('no-async-actions', rule, {
+  valid: [
+    {
+      code: `
+      Component.extend({
+      actions: {
```

Comment:
This is unreadable without proper indentation.

Reply:
Indented propertly